packages/core/src/main/notification/NotificationService.ts

```diff
@@ -0,0 +1,59 @@
+import {CRUDEngine} from '@wireapp/store-engine/dist/commonjs/engine/index';
```

Comment:
```js
import {CRUDEngine} from '@wireapp/store-engine/dist/commonjs/engine/';
```

Reply:
We decided to explicitly import `index` because of compatibility issues. Later on we will move to `root` instead of `index` to enforce it.

Reply:
With our new export syntax we might be able to use `index.ts`. We should check that once more because this is common sense: https://basarat.gitbooks.io/typescript/docs/tips/barrel.html